### PR TITLE
💽 feat: Add CONSOLE_JSON for deploying to GCP K8S env

### PR DESCRIPTION
**Pull request by idachev**
# Add json console logging

## Summary

Added CONSOLE_JSON to start dump the logs as UTC timestamp and as json.

## Change Type

Please delete any irrelevant options.

- New feature (non-breaking change which adds functionality)

## Testing
Before:
image

After:
image

## Checklist

Please delete any irrelevant options.

-  My code adheres to this project's style guidelines
-  I have performed a self-review of my own code
-  I have commented in any complex areas of my code
-  I have made pertinent documentation changes
-  My changes do not introduce new warnings

### CODE REVIEW

**Comment by danny-avila** [comment]
I noticed you added docker build scripts, can you explain them a bit? I don't have issues with including but noticed you didn't mention them.

Thanks for the PR!

**Comment by idachev-shipcars** [comment]
> I noticed you added docker build scripts, can you explain them a bit? I don't have issues with including but noticed you didn't mention them.
> 
> Thanks for the PR!

Yep, I use these to build the docker and push it to the custom Google GCP docker registry. You can pass arguments where to push. I know there are scripts to do this in GitHub Actions, but these are for the official repo and cannot be easily used to push to a custom registry.

**Review by danny-avila — APPROVED** [review]

File: package-lock.json
Comment (danny-avila) [inline package-lock.json:1]: If you can, remove package-lock.json from the PR. I believe you might have to use `git reset`.

**Review by danny-avila — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (idachev) [reply package-lock.json:1]: Removed

**Review by danny-avila — APPROVED** [review]

**Pull request merged**